test/integration_test.js

```diff
@@ -698,6 +698,15 @@ describe("Integration tests", function() {
                   }, {safe: true}))
                   .should.be.rejectedWith(Error, /412 Precondition Failed/);
               });
+
+              it("should not override existing data with no last_modified", () => {
+                return coll.createRecord({title: "foo"})
+                  .then(({data}) => coll.updateRecord({
+                    id: data.id,
+                    title: "foo",
+                  }, {safe: true}))
+                  .should.be.rejectedWith(Error, /412 Precondition Failed/);
+              });
```

Comment:
We should add a test to ensure a non-existent resource is created when safe is true.